acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -53,103 +64,106 @@ describe('Critical user path(s)', () => {
     });
 
     /**
-     * Navigate from Who do you work for? page
+     * Run a sequence of actions to simulate verification of a user
      *
-     * @returns {void}
+     * @returns {Promise}
      */
-    async function whoDoYouWorkForPage() {
-        await page.waitForSelector(ORGANISATION_INPUT);
+    async function verifyUser() {
+        // start
+        await clickElement(CONTINUE_BUTTON);
+        // who-do-you-work-for
         await page.$eval(ORGANISATION_INPUT, (element) => {
```

Comment:
The element.value cannot be overwritten by a nodeJS variable as the context in which this section of code will be run is in the browser and has no access to the node variables (the $eval function)

hence the string will need to be stated explicitly
@sulthan-ahmed 

other functions  have been converted to functions